cpp/python/src/notifier.cpp

```diff
@@ -23,16 +23,17 @@ Notifier::~Notifier() {}
 
 void Notifier::scheduleFutureNotify(std::shared_ptr<::ucxx::Future> future, ucs_status_t status)
 {
-  ucxx_trace_req(
-    "Notifier::scheduleFutureNotify(): future: %p, handle: %p", future.get(), future->getHandle());
+  ucxx_trace_req("ucxx::python::Notifier::scheduleFutureNotify, future: %p, handle: %p",
+                 future.get(),
+                 future->getHandle());
   auto p = std::make_pair(future, status);
   {
     std::lock_guard<std::mutex> lock(_notifierThreadMutex);
     _notifierThreadFutureStatus.push_back(p);
     _notifierThreadFutureStatusReady = true;
   }
   _notifierThreadConditionVariable.notify_one();
-  ucxx_trace_req("Notifier::scheduleFutureNotify() notified: future: %p, handle: %p",
+  ucxx_trace_req("ucxx::python::Notifier::scheduleFutureNotify, notified: future: %p, handle: %p",
```

Comment:
```suggestion
  ucxx_trace_req("ucxx::python::Notifier::scheduleFutureNotify, notified future: %p, handle: %p",
```

Reply:
Good catch, fixed in https://github.com/rapidsai/ucxx/pull/156/commits/2f4317c5b0fd200669cfbc821d54c48dde98a263 .